src/components/Book.js

```diff
@@ -2,20 +2,39 @@ import React from 'react';
 import { Row, Col, Button } from 'react-bootstrap';
 import { PropTypes } from 'prop-types';
 import { useDispatch } from 'react-redux';
+import { CircleProgress } from 'react-gradient-progress';
 import { removeBook } from '../redux/books/books';
 
 export default function Book(props) {
   const dispatch = useDispatch();
   const { data } = props;
   return (
-    <div>
-      <Row className="m-5 border border-2 bg-white">
+    <div className="container">
+      <Row className="mt-4 border border-2 bg-white">
+        <Col className="p-4 ">
+          <p className="text-secondary">{data.category}</p>
+          <h5 className="fw-bold m-0">{data.title}</h5>
+          <p className="text-secondary">Robert</p>
+          <ul className="d-flex list-unstyled gap-3 m-0">
+            <li><button type="button" className="li-button">comments </button></li>
+            <li><button type="button" className="li-button" onClick={() => dispatch(removeBook(data))}>remove </button></li>
+            <li><button type="button" className="li-button">edit </button></li>
+          </ul>
```

Comment:
- [ ] Kindly make sure that your styling is matching the design as much as possible, currently, these elements are not matching the color found in the Zeplin design, also don't forget to add the separators `|` for user actions.
Your styling
![image](https://user-images.githubusercontent.com/33767951/140619771-17d68de1-c461-4df6-8cd0-cc937d99c96b.png)
Zeplin design
![image](https://user-images.githubusercontent.com/33767951/140619826-69dd58c8-46d5-4bb2-be00-f3d4b889959f.png)